packages/webapp/src/containers/Setup/SetupLeftSection.tsx

```diff
@@ -38,11 +38,11 @@ function SetupLeftSectionFooter() {
  * Setup left section header.
  */
 function SetupLeftSectionHeader() {
-  const { setLogout } = useAuthActions();
+  const { mutateAsync: oidcLogoutMutate } = useAuthOidcLogout();
 
   // Handle logout link click.
   const onClickLogout = () => {
-    setLogout();
```

Comment:
@abouolia I have created custom logout hook for oidc(useAuthOidcLogout) where setLogout will be called after the oidc logout api call in both condition (error or success). Is it ok to follow this?